dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/web/ContentletWebAPIImpl.java

```diff
@@ -423,11 +423,12 @@ private void saveWebAsset(Map<String, Object> contentletFormData,
 			}
 		 }
 
-		if (categoriesList != null && categoriesList.size() > 0) {
-			for (final Iterator iterator = categoriesList.iterator(); iterator.hasNext();) {
-
-				final String tmpString = (String) iterator.next();
-				categories.add(catAPI.find(tmpString, user, false));
+		if (UtilMethods.isSet(categoriesList)) {

```

Comment:
Should the condition be:

```
if (UtilMethods.isSet(categoriesList) && categoriesList.size() > 0){
```

Reply:
@wezell the `isSet` method already checks for both null and empty list scenarios:
```java
public static final boolean isSet(final Collection<?> collection) {
    return null != collection && !collection.isEmpty();
}
```